.eslintrc.js

```diff
@@ -192,6 +192,30 @@ module.exports = {
                 "@typescript-eslint/no-unused-expressions": "off",
             },
         },
+        {
+            files: ["*.ts", "*.tsx"],
+            parser: "@typescript-eslint/parser",
+            parserOptions: {
+                project: [
+                    "tsconfig.json",
+                    "packages/*/tsconfig.json",
+                    "packages/tsconfig-shared.json",
```

Comment:
We can simplify this a bit.  The second two lines bring in TS Config files that aren't needed (the root `tsconfig.json` can be thought of as our dev environment TypeScript config and covers all source files (except Cypress).  

```suggestion
                    "tsconfig.json",
```